code/modules/clothing/under/miscellaneous.dm

```diff
@@ -867,3 +867,30 @@
 	body_parts_covered = CHEST|GROIN|LEGS|FEET
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
+
+/obj/item/clothing/under/contortionist
+	name = "atmospheric technician's jumpsuit"
+	desc = "A light jumpsuit useful for squeezing through narrow vents."
+	icon_state = "atmos"
+	item_state = "atmos_suit"
+	item_color = "atmos"
+	resistance_flags = FIRE_PROOF
+
+/obj/item/clothing/under/contortionist/equipped(mob/living/carbon/human/user, slot)
+	if(!user.ventcrawler)
+		user.ventcrawler = 2
+	..()
+
+/obj/item/clothing/under/contortionist/dropped(mob/living/carbon/human/user)
+	if(!user.get_int_organ(/obj/item/organ/internal/heart/gland/ventcrawling))
+		user.ventcrawler = 0
+	..()
+
+/obj/item/clothing/under/contortionist/proc/check_clothing(mob/user as mob)
+	//Allowed to wear: glasses, shoes, gloves, pockets, mask, and jumpsuit (obviously)
+	var/list/slot_must_be_empty = list(slot_back,slot_handcuffed,slot_legcuffed,slot_l_hand,slot_r_hand,slot_belt,slot_head,slot_wear_suit)
```

Comment:
back, handcuffed, legcuffed, l_store, r_store, head, wear_suit, belt
see code/modules/mob/living/carbon/human/inventory.dm for why